src/read.c

```diff
@@ -4622,7 +4622,7 @@ static avifResult avifParse(avifDecoder * decoder)
 #endif
     avifBool needsTmap = AVIF_FALSE;
     avifBool tmapSeen = AVIF_FALSE;
-    avifFileType ftyp = {};
+    avifFileType ftyp = {0};
```

Comment:
@hmaarrfk, my C isn't great. Is it required to use 

```suggestion
    avifFileType ftyp = { {0, 0, 0, 0}, {0, 0, 0, 0}, NULL, 0 };
```

instead of just `{0}`?

Reply:
i had the same thought to try `{0}` lets see if the conda-forge CIs pass

Reply:
happy to report your 1 character changed fixed things at conda-forge

Reply:
https://github.com/conda-forge/libavif-feedstock/pull/40/checks?check_run_id=38067816876